src/components/Answer.tsx

```diff
@@ -1,9 +1,31 @@
 import cn from 'classnames';
 import { ReactNode } from 'react';
 import { ALPHABET, isAnswerChecked, isAnswerCorrect, isAnswerIncorrect } from '../utils';
-import { Answer as AnswerType, ChosenAnswer, ID } from '../types';
+import { Answer as AnswerType, ID } from '../types';
 import { Content } from './Content';
 import { SimpleFeedback } from './Feedback';
+import styled from 'styled-components';
+import { colors } from '../theme';
+
+const StyledAnswerIndicator = styled.div<{ state: boolean }>`
+  color: ${props => props.state ? colors.answer.correct : colors.answer.incorrect};
+  text-transform: uppercase;
+  font-size: 1.1rem;
+  font-weight: bold;
+`;
+
+const AnswerIndicator = (
+  { isCorrect, isIncorrect }: { isCorrect?: boolean; isIncorrect?: boolean }
+) => {
+  if (!isCorrect && !isIncorrect) {
```

Comment:
Does this component also have to disappear when you select a new answer or can it stay?

Reply:
It needs to stay for accessibilty